src/conf_mode/dns_dynamic.py

```diff
@@ -86,17 +89,20 @@ def verify(dyndns):
             if field not in config:
                 raise ConfigError(f'"{field.replace("_", "-")}" {error_msg_req}')
 
-        # If dyndns address is an interface, ensure that it exists
+        # If dyndns address is an interface, ensure
+        # that the interface exists (or just warn if dynamic interface)
         # and that web-options are not set
         if config['address'] != 'web':
             # exclude check interface for dynamic interfaces
-            interface_filter = ('pppoe', 'sstpc')
-            if config['address'].startswith(interface_filter):
-                Warning(f'interface {config["address"]} does not exist!')
+            if config['address'].startswith(dynamic_interfaces):
+                Warning(f'Interface "{config["address"]}" does not exist for Dynamic DNS '
+                        f'service "{service}" yet! The service will not operate correctly '
+                        f'till the interface is configured.')
```

Comment:
The statement is incorrect. It should be:

```
'Interface "{config["address"]}" does not yet exist and con not be used for Dynamic DNS '
f'service "{service}" until it is up!')
```


PPPoE interfaces might be configured but not up as the BRAS connection has not yet been established or is currently disturbed.

Reply:
Message updated.